x-pack/plugins/monitoring/server/lib/metrics/apm/metrics.js

```diff
@@ -187,6 +187,12 @@ export const metrics = {
     label: 'Metric',
     description: 'Metric events processed'
   }),
+  apm_responses_errors_internal: new ApmEventsRateClusterMetric({
```

Comment:
Mind moving this to be with the other errors? Needs to be retested anyway.

Reply:
good call.  It's was grouped in the original but 6.5 doesn't have the `i18n.translate` bits so backporting was a bit fuzzy.  updated now.